cmd/restic/cmd_backup.go

```diff
@@ -520,6 +525,19 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 	}
 
 	var targetFS fs.FS = fs.Local{}
+	if runtime.GOOS == "windows" && opts.UseWindowsVSS {
+
+		if !fs.HasSufficientPrivilegesForVSS(){
+			return errors.Fatal(
+				"user doesn't have sufficient privileges to use VSS " +
+				"snapshots\n",
```

Comment:
I'd suggest merging this with the previous line so it's just one line.

Reply:
I kept this one to stay below 80 characters per line. Any official statement concerning this one? I know there are other lines that are longer. ;-)

Reply:
In this case, please join the lines, it's not a problem that it exceeds 80 chars :) Much cleaner that way.